online_meetings/canvas_zoom_meetings.py

```diff
@@ -0,0 +1,139 @@
+# Script to get all sites where Zoom is visible and retrieve the meetings to generate a report
+
+import json
+import yaml
+import re
+import os
+import sys
+import requests
+import logging
+import http.client
+from datetime import datetime
+
+from canvasapi import Canvas
+from bs4 import BeautifulSoup as bs
+
+import pandas as pd
+
+# read configurations
+try:
+    with open(os.path.join('../config', 'env.json')) as env_file:
+        ENV = yaml.load(env_file.read())
+except FileNotFoundError:
+    sys.exit(
+        'Configuration file could not be found; please add env.yaml to the config directory.')
```

Comment:
Let's just update this to `env.json` for the moment?

Reply:
I changed the file name but missed the exit message, will update this.